### DISCUSSION TITLE
求支付宝续费链接

### DISCUSSION THREAD

**Discussion by fuyujie8810** [getlantern/lantern-forum-cn#915] [Q&A]
fuyujie8810@163.com

**Comment by MaxMKMax** [comment]
@fuyujie8810 您好. 好的, 已经帮您找支付部门要支付链接了. 请您稍等. 谢谢您支持蓝灯!

**Comment by fuyujie8810** [comment]
***@***.***，然后来信告诉我也可以。


---- 回复的原邮件 ----
| 发件人 | Mike ***@***.***> |
| 发送日期 | 2024年10月2日 02:23 |
| 收件人 | ***@***.***> |
| 抄送人 | ***@***.***> ,
***@***.***> |
| 主题 | Re: [getlantern/lantern-forum-cn] 求支付宝续费链接 (Discussion #915) |

@fuyujie8810 您好. 好的, 已经帮您找支付部门要支付链接了. 请您稍等. 谢谢您支持蓝灯!

**Comment by MaxMKMax** [comment]
@fuyujie8810 
您好. 这是您的专属购买链接. 谢谢您支持蓝灯.
https://buy.stripe.com/fZebJi5sf8y2fte289
